Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.py

```diff
@@ -829,14 +906,24 @@ def fetch_incidents(client, first_fetch_time, integration_instance, last_run: di
     try:
         # The count of incidents, so as not to pass the limit
         count_incidents = 0
-
+        incident_ids: list[str] = []
+        incident_data_dict: dict[str, Any] = {}
+        if_upgraded_extra_data = False
+        if raw_incidents and (check_using_upgraded_api_incidents_extra_data(client, raw_incidents[0].get('incident_id')[-1])):
+            incident_ids = [raw_incident.get('incident_id') for raw_incident in raw_incidents
+                            if len(incident_ids) < max_fetch]
+            raw_incidents_data, if_upgraded_extra_data = check_using_upgraded_api_incidents_extra_data(client, incident_ids)
```

Comment:
Why not using the `get_multiple_incidents_extra_data` instead this method? this method is meant to be used only to know if to use the new API endpoint